mkosi/manifest.py

```diff
@@ -197,6 +199,31 @@ def record_deb_packages(self, root: Path) -> None:
 
             source_package.add(package)
 
+    def record_pkg_packages(self, root: Path) -> None:
+        package_descriptions = [str(p) for p in Path(f"{root}/var/lib/pacman/local").glob("*/desc")]
```

Comment:
```suggestion
        package_descriptions = [str(p) for p in (root / "var/lib/pacman/local").glob("*/desc")]
```
or
```suggestion
        package_descriptions = [str(p) for p in root.joinpath("var/lib/pacman/local").glob("*/desc")]
```
since root already comes in as `Path`.

Reply:
The slashes are nicer